### DISCUSSION TITLE
Incorporate Q

### DISCUSSION THREAD

**Discussion by astehno** [coastal-hazards-system/StormSim-Library#87] [Announcements]
For RB3, add Q.
1. Keep q as an output. Also, in EurOtop code, output q_waves and q_overflow separate from q - need to change line 275 for this.
2. Output Q(t) to show the accumulation of q over time and also output Q(lastTimestep) as the final volume.
3. How do we want to handle Q(t) hazard? If we don't report as hazard, how do we report? If we don't report at all, ALS wants to use this data for some research so just give it to her.

**Comment by fgarcia0128** [comment]
Current implementation:
- Q has been incorporated for RB3. The way is computed is by summing q(t) for each storm. The resulting matrix (nStorms x Replicates) is passed through SST/JPM.

**Comment by fgarcia0128** [comment]
Implemented
